.coafile

```diff
@@ -0,0 +1,72 @@
+[sql]
+bears = InvalidLinkBear
+files = **.sql
+
+[volt]
+bears = InvalidLinkBear, LineLengthBear, SpaceConsistencyBear
+files = **.volt
+max_line_length = 200
+use_spaces = True
+enforce_newline_at_EOF = True
```

Comment:
this, and `use_spaces`, should be in the Default config section, so they do not need to be repeated.